Source/iop/IopBios.cpp

```diff
@@ -50,8 +50,11 @@
 #define BIOS_MODULESTARTREQUEST_HEAD_BASE (CIopBios::CONTROL_BLOCK_START + 0x0018)
 #define BIOS_MODULESTARTREQUEST_FREE_BASE (CIopBios::CONTROL_BLOCK_START + 0x0020)
 #define BIOS_HANDLERS_BASE (CIopBios::CONTROL_BLOCK_START + 0x0100)
-#define BIOS_HANDLERS_END (BIOS_THREADS_BASE - 1)
-#define BIOS_THREADS_BASE (CIopBios::CONTROL_BLOCK_START + 0x0200)
+#define BIOS_HANDLERS_END (BIOS_PSEUDO_INTRHANDLER_TABLE_BASE - 1)
+#define BIOS_PSEUDO_INTRHANDLER_TABLE_BASE (0x480)
+#define BIOS_PSEUDO_INTRHANDLER_ENTRY_SIZE (4 * 2)

```

Comment:
Please add a structure named `SYSTEM_INTRHANDLER` with the two members instead of defining the size like that.